site/docs/v1/tech/apis/system.adoc

```diff
@@ -415,21 +415,25 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 [method]#GET# [uri]#/api/prometheus/metrics#
 --
 
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#basic-authentication[icon:shield[type=fas]] Retrieve FusionAuth application metrics to use with Prometheus.
```

Comment:
This is fine if you prefer this. 

However, since the description is the same, and the endpoint is the same, we could also just add the two icons next to each there so both red icons would show up.

Also, if we wanted to differentiate these two red icons just a touch more...  `[icon:shield-alt[type=fas]]` looks pretty good. You could tweak the colors on it to look closer to our red icons.

Reply:
Love the two icons next to each other. Shield-alt does look better. Will change to that.